extensions/amp-embedly-card/1.0/amp-embedly-key.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Layout} from '#core/dom/layout';
+import {PreactBaseElement} from '#preact/base-element';
+import {isExperimentOn} from '#experiments';
+import {userAssert} from '../../../src/log';
+
+/** @const {string} */
+export const TAG = 'amp-embedly-key';
+
+export class AmpEmbedlyKey extends PreactBaseElement {
+  /** @override */
+  isLayoutSupported(layout) {
+    userAssert(
+      isExperimentOn(this.win, 'bento') ||
+        isExperimentOn(this.win, 'bento-embedly-key'),
```

Comment:
```suggestion
        isExperimentOn(this.win, 'bento-embedly-card'),
```

Let's have this use the same experiment as the `amp-embedly-card` component for smoother devx - should be safe to assume that enabling one experiment can imply the api key feature for this component.